lrpar/src/lib/ctbuilder.rs

```diff
@@ -691,16 +670,17 @@ C : 'a';"
                 .as_bytes()
         );
 
-        match CTParserBuilder::new()
+        let mut ct = CTParserBuilder::new()
             .error_on_conflicts(false)
-            .action_kind(ActionKind::GenericParseTree)
-            .process_file_in_src(file_path.to_str().unwrap())
-        {
-            Ok((_, conflicts)) => {
-                assert_eq!(conflicts.as_ref().unwrap().conflicts().sr_len(), 1);
-                assert_eq!(conflicts.as_ref().unwrap().conflicts().rr_len(), 1);
+            .action_kind(ActionKind::GenericParseTree);
+        let _ = ct.process_file_in_src(file_path.to_str().unwrap());
+
+        match ct.conflicts() {
+            Some((_, _, _, conflicts)) => {
```

Comment:
The `_`s are grm, sgraph, stable respectively?

Reply:
Yes, I was thinking about putting `_grm`, etc, but last time I did this you told me `_` is enough.

Reply:
I didn't mean you should change them: I just wanted to check.